contracts/strategy/BaseStrategy.sol

```diff
@@ -374,13 +361,30 @@ abstract contract BaseStrategy is IStrategy, AccessControl {
             return 0;
         }
 
-        uint256 exchangeRate = exchangeRateFeeder.exchangeRateOf(
-            address(ustToken),
-            true
-        );
+        uint256 aUstPrice = _aUstExchangeRate();
 
         return
-            ((exchangeRate * aUstBalance) / 1e18) -
-            _performanceUstFeeWithInfo(aUstBalance, exchangeRate);
+            ((aUstPrice * aUstBalance) / _aUstToUstFeedDecimals) -
+            _performanceUstFeeWithInfo(aUstBalance, aUstPrice);
+    }
+
+    /**
+     * @return aUST / UST exchange rate from chainlink
+     */
+    function _aUstExchangeRate() internal view virtual returns (uint256) {
```

Comment:
can we call this `aUstToUstExchangeRate`?
because on the `NONUstStrategy` the name makes it seem as if it's `aUst / underlying`, which is not true

Reply:
agree

Reply:
fixed